NEWS.rst

```diff
@@ -1,3 +1,10 @@
+3.6.4 (not yet released)
```

Comment:
Thanks Sascha!

Could you please remove the custom header in the changelog? Just add your new entries without leaving a blank line at the top. The version string, timestamp, dash titling and newlines are automatically generated by the release script. See inside the setup.py, the `release` command class.